cypress/support/step-definitions/step-sequence-steps.js

```diff
@@ -22,6 +23,10 @@ Then('hidden label is set to {word}', (hiddenLabel) => {
   ariaLabel(ARIA_LABEL).contains(hiddenLabel).should('have.text', hiddenLabel);
 });
 
+Then('label {string} color is set to {string}', (text, color) => {
```

Comment:
and remove please also `orientation is set to {string}` it is not used anymore :)